### ISSUE TITLE
ui.TextField - become_first_responder

### ISSUE THREAD

**Issue by zrzka** [omz/Pythonista-Issues#393]
Would be nice to be able to call it directly on `ui.TextField`. Now, I can do it via `objc_util`, checking subviews, searching for `UITextField` and calling `becomeFirstResponder()` on it. Not a big deal for me, can workaround it, just don't like that I do rely on Pythonista internals and it can break (not likely) in the future.

**Comment by dgelessus** [comment]
Is the `begin_editing()` method what you are looking for? (I think it's still undocumented though.) I'm not very familiar with UIKit, so I don't know if "first responder" means anything other than "has focus".

**Comment by zrzka** [comment]
Yup, `becomeFirstResponder()` means get focus, show keyboard, show cursor and user can start typing. `begin_editing()` does the same thing, thanks! Okay, so this issue is about missing `begin_editing` in docs. Going to update title ...
